pkg/agent/agent.go

```diff
@@ -231,14 +226,18 @@ func (a *Agent) bootstrap() error {
 		a.baseSVIDKey = key
 
 		// If we're here, we need to attest/Re-attest
-		regEntryMap, err := a.attest()
-		if err != nil {
-			return err
-		}
-		err = a.FetchSVID(regEntryMap, a.BaseSVID, a.baseSVIDKey)
+		regEntries, err := a.attest()
 		if err != nil {
 			return err
 		}
+		uri := a.config.TrustDomain
+		uri.Path = "spiffe/cp"
+		cacheManager := manager.NewManager(
```

Comment:
Maybe this should be in cache package? It would then be `cache.NewManager`

Reply:
makes sense will change the package